taskfiles/utils-cpp-lint.yaml

```diff
@@ -6,34 +6,284 @@ shopt: ["globstar"]
 tasks:
   # Runs clang-format on C++ files at the given paths.
   #
-  # @param {string} FLAGS Any flags to pass to clang-format.
-  # @param {string[]} SRC_PATHS The paths on which to run clang-format.
-  # @param {string} VENV_DIR Python virtual environment where clang-format is installed.
+  # @param {string[]} ROOT_PATHS The paths on which to start `find`. When combined with
+  # EXCLUDE_PATTERNS and INCLUDE_PATTERNS `find` will output a list of files used as the input to
+  # `clang-format`.
+  # @param {string} VENV_DIR Python virtual environment where `clang-format` is installed.
+  # @param {string[]} [EXCLUDE_PATTERNS] `-path` wildcard patterns to prune in `find`.
+  # @param {string[]} [EXTENSION_PATTERNS=["*.cpp", "*.h", "*.hpp"]] `-name` wildcard patterns to
+  # match in `find`.
+  # @param {string} [FLAGS] Any flags to pass to `clang-format`.
+  # @param {string[]} [INCLUDE_PATTERNS] `-path` wildcard patterns to match in `find`.
```

Comment:
* Can we remove `find` implementation details?
* Can we reorder the arguments to be more logical? Perhaps:
  * `VENV_DIR`
  * `ROOT_PATHS`
  * `INCLUDE_PATTERNS`
  * `EXCLUDE_PATTERNS`
  * `EXTENSION_PATERNS`
  * `FLAGS`